go.mod

```diff
@@ -107,7 +108,7 @@ require (
 	google.golang.org/appengine v1.6.7 // indirect
 	google.golang.org/genproto v0.0.0-20230110181048-76db0878b65f // indirect
 	google.golang.org/grpc v1.53.0 // indirect
-	google.golang.org/protobuf v1.28.1 // indirect
```

Comment:
Same here, seems like because of this change, you need to run `go mod vendor` that will generate new content in `vendor` folder, then you need commit it to avoid failure of `check-commited-changes` job.